scripts/help.sh

```diff
@@ -63,8 +63,9 @@ function features_usage {
   echo "  --enable_pkt_translation:     enable packet translation                  (default: false)"
   echo "  --enable_pkt_drop:            enable packet drop                         (default: false)"
   echo "  --enable_rtt:                 enable RTT tracking                        (default: false)"
-  echo "  --enable_udn_mapping:         enable User Defined Network mapping (default: false)"
+  echo "  --enable_udn_mapping:         enable User Defined Network mapping        (default: false)"
   echo "  --get-subnets:                get subnets information                    (default: false)"
+  echo "  --sampling:                   rate at which packets should be sampled    (default: 1)"
```

Comment:
not sure how much text we can write here without being too verbose, but when referring to sampling I like to be more accurate about what it really does, because it's not obvious for newcomers. E.g: "a value of 50 means one packet every 50 will be sampled".

Also: it would be better to talk about sampling `ratio` rather than `rate` (but this remark applies to everywhere, not just here/CLI) - see [here](https://mathmonks.com/ratio/ratios-and-rates) for the difference. A sampling rate would generally mean how many packets per second are sampled, whereas a sampling ratio is what we want here - the proportion of packets that we keep.
I'll open PRs in other parts to do this change.

Reply:
See https://github.com/netobserv/network-observability-operator/pull/1342

Reply:
> That description should remain short but we can include this option in an example and put the full sentence to explain it 👌
> 
> In that case we should write the `ratio divisor` as the `dividend` is forced to 1 in our usage ?

Technically true but that makes the sentence quite more complicated! I think we can make it easy to understand without being too pedantic (that's why a short example helps IMO). Another way to put it would be: "value that defines the ratio of packets being sampled".

Reply:
done in https://github.com/netobserv/network-observability-cli/pull/209/commits/ba6ec93024322160997b6373e5a0e1ea1d39989e